app/controllers/vouchers_controller.rb

```diff
@@ -0,0 +1,11 @@
+class VouchersController < ApplicationController
+  before_action :authenticate_retailer!
+  load_and_authorize_resource
+  def index
+    @vouchers
```

Comment:
This, thanks to the [Ability set up here](https://github.com/bylotech/api-billicopresto/pull/17/files#diff-0fdc2ce2595c4fe479d6a97b404013651e8a0e2d4b02d93c87843b3a30103d9fR11), automatically loads all the vouchers related to the Retailer currently logged in